### PR TITLE
OPENEUROPA-469: Add first draft of language switcher block.

### PR DESCRIPTION
**Pull request by imanoleguskiza**
### Description

Add custom language selector block that shows available translations when the selected language is not available

### Change log

- Added: Add custom language selector block

### CODE REVIEW

File: config/optional/block.block.oe_content_language_switcher.yml [hunk 1]
```diff
@@ -0,0 +1,19 @@
+langcode: en
+status: true
+dependencies:
+  module:
+    - oe_multilingual
+  theme:
+    - oe_theme
+id: oe_content_language_switcher
+theme: oe_theme
+region: content
```
Comment (brummbar) [inline config/optional/block.block.oe_content_language_switcher.yml]: By looking at the [specifications of the ECL component](https://ec-europa.github.io/europa-component-library/ec/components/detail/ec-component-lang-select-page--default) this component should go under the breadcrumbs. So it should be in the breadcrumbs region. Note that also the primary color should be fixed.

File: oe_multilingual.module [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+ * will return all available languages and then choose the first one
+ * ordered alphabetically.
+ * - Return the entity's original language as the default fallback language.
+ */
+function oe_multilingual_language_fallback_candidates_alter(array &$candidates, array $context): void {
+  $operation = $context['operation'];
+
+  // The default fallback language for any entity is the language it was
+  // originally written on.
+  if ($operation == 'entity_upcast' || $operation == 'entity_view') {
```
Comment (brummbar) [inline oe_multilingual.module]: Are we sure these are the only operations possible?
Also, use the identical operator.

File: config/optional/block.block.oe_content_language_switcher.yml (hunk 1 above)
Comment (imanoleguskiza) [reply config/optional/block.block.oe_content_language_switcher.yml]: The problem with this is that the current ecl implementation has a background color so it doesn't much up at all if placed in the breadcrumbs or navigation regions. The only place where it doesn't stick out like a sore thumb is on the content region. I propose we solve this in a follow up ticket

File: oe_multilingual.module (hunk 2 above)
Comment (imanoleguskiza) [reply oe_multilingual.module]: This are the two operations that are involved in the rendering of an entity. For the defined use case we only need to alter the language fallback when rendering nodes.

**Comment by brummbar** [comment]
After merging master back, tests failed due to changes in language labels. Updated the test, ready for review.

**Pull request merged**